docs/resources/servicestage_component_instance.md

```diff
@@ -59,6 +59,16 @@ resource "huaweicloud_servicestage_component_instance" "default" {
       name  = "TZ"
       value = "Asia/Shanghai"
     }
+
+    log_collection_policy {
+      host_path = "/tmp"
+
+      container_mounting {
+        path             = "/attached/01"
+        host_extend_path = "None"
```

Comment:
suggest add the reason why the value of `host_extend_path` is None

Reply:
provides a list of enumerated values.